pkg/webservice/handlers.go

```diff
@@ -263,6 +281,44 @@ func getNodeJSON(nodeInfo *cache.NodeInfo) *dao.NodeDAOInfo {
 	}
 }
 
+func getNodesUtilJSON(partition *cache.PartitionInfo, name string) *dao.NodesUtilDAOInfo {
+	mapResult := make([]int, 10)
+	mapName := make([][]string, 10)
+	var v float64
+	var nodeUtil []*dao.NodeUtilDAOInfo
+	for _, node := range partition.GetNodes() {
+		total, ok := node.GetCapacity().Resources[name]
+		if !ok {
+			total = 0
+		}
+		resourceAllocated, ok := node.GetAllocatedResource().Resources[name]
+		if !ok {
+			resourceAllocated = 0
+		}
+		if float64(total) > 0 {
+			v = float64(resourceAllocated) / float64(total)
+		} else {
+			v = float64(0)
+		}
```

Comment:
Float divisions are 0 safe in go. It will return + or - inf. If the total resource is 0, but we have some usage, 0 is returned, what am not sure it is OK, because that means there is some misconfiguration/error and we will return that we have no usage. I would rather return N/A or "Inf", because we cannot calculate it

Reply:
In resources.go file there is already a method for calculating absolute usage: func CalculateAbsUsedCapacity(capacity, used *Resource) *Resource please use that one if possible.

Reply:
done